src/main/java/com/mycompany/hangman/actions/ExitAction.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *   To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.mycompany.hangman.actions;
+
+import java.awt.event.ActionEvent;
+import javax.swing.AbstractAction;
+
+/**
+ *
+ * @author Cory
+ */
+public class ExitAction extends AbstractAction
+{
+
+    public ExitAction()
+    {
+        super("Exit");
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e)
+    {
+           System.exit(0);

```

Comment:
there’s a bit of overlap between what action classes do and what a controller is supposed to do (both take user input and affect the model accordingly). the “new game” action simply relays the desired action to the controller but this one immediately shuts down the app. i’d suggest making this simply call some sort of userRequestedExit() on the controller. the controller should be notified in case it wants to save settings/state or shutdown resources, etc

Reply:
I think I will eventually move into this direction.